src/components/EventOverview/EventsPage.tsx

```diff
@@ -102,19 +127,29 @@ const HomeLandingPage = () => {
   );
 
   return (
-    <div className="landing-wrapper">
-      <div className="landing-top-section">
-        <div className="landing-top-bar">
+    <Box className={classes.root}>
+      <Box className={classes.topSection}>
+        <Box className={classes.topBar}>
           <Typography variant="h3">Mass Casualty Events</Typography>
+          {/* <Box display="flex" alignItems="center">
+            <Typography
+              variant="h6"
+              align="right"
+              style={{ marginRight: '16px' }}
+            >
+              Joe Li
+            </Typography>
+            <AccountCircleIcon fontSize="large" color="secondary" />
+          </Box> */}
```

Comment:
guessing this can be replaced for `UserProfile`
cc: @JasonYG 
```suggestion
```

Reply:
Yup, sorry I meant to remove this but I had initially commented it out in case I resolved conflicts incorrectly. Thanks!